### PR TITLE
Bump version to 0.6.2

### PR DESCRIPTION
**Pull request by ultraviolet-1986**
Rebuild to newer version (0.6.2) released 7 days ago (as of this commit).

### CODE REVIEW

**Comment by popey** [comment]
Thanks so much! 
This is currently blocked by #3 
I'll take a look at get that fixed.Thanks!

**Comment by ultraviolet-1986** [comment]
Hi, looks like this PR can be closed. The error appears to be with CorsixTH source itself. I've reported the issue at: <https://github.com/CorsixTH/CorsixTH/issues/1433>. Thought triggering another build would be enough but trying it myself led down a rabbit hole. I'll keep tinkering to see what I can come up with. Thanks again!

**Comment by TheCycoONE** [comment]
What error?

The default layout was changed to better conform to the FHS as reflected in #3.

**Comment by ultraviolet-1986** [comment]
Hi, my train of thought was a bit of a dead end (I am pretty new to this so I'll apologise here) - building from source only *appeared* to fail, and the executable would not start. As pointed out here <https://github.com/CorsixTH/CorsixTH/issues/1433>, it looks like simply building and launching is no longer enough to play. Building and *then* running `sudo make install` installs the game proper and allows it to play perfectly.

TL;DR There is no error, the tarball requires building and installing rather than building and playing.

Tested and confirmed using Ubuntu 16.04 and 18.04 respectively. Sorry for dragging us into the weeds.

**Comment by TheCycoONE** [comment]
To clarify what's going on; in 0.61 and prior versions the executable would do a search for CorsixTH.lua from it's current directory up to the root. This was incompatible with the FHS, since game resources are expected to be under /usr/share/game-name, and executables are expected to be under /usr/bin

Typically distributions would work around this by putting everything in one place /usr/share/game-name, or /opt/game-name, and in /usr/bin add a shell script to launch the game.

With 0.62, we no longer search down to the root, but look in a single location determined at compile time (there is a run time override flag, but generally a package shouldn't use it.) By default on Linux and other non Mac-OS Unix systems the default location checked for data files is /usr/share/corsix-th/, and the binary produced is now called corsix-th.

Specifically we use [GNUInstallDirs](https://cmake.org/cmake/help/v3.10/module/GNUInstallDirs.html ) to determine the directory, so to use a base path other than /usr, change CMAKE_INSTALL_PREFIX, and to change the location of the data files specifically use CMAKE_INSTALL_DATADIR.

USE_SOURCE_DATADIRS is a flag added for developers so that the game resources from the source directory will be used. It is incompatible with distribution.

I hope that helps.
